### PR TITLE
feat:  implement display mode buttons

### PR DESCRIPTION
**Pull request by eddieantonio**
~~Add a toggle for "basic" and "traditional" mode. Basic is what is up now; "traditional" mode will display something like "Noun (NAD-4)" instead of emoji on the search page.~~

EDIT: there are two modes: **community** and **linguistic**. The toggles are tucked into the bottom of the screen. Turn on "lingusitic mode" and search results change their display to this:

mîciw, linguistic mode

Compare with the default:

mîciw, community mode

Start to address #708

### CODE REVIEW

File: CreeDictionary/CreeDictionary/display_options.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+"""
+Display options. Currently controls:
+
+ - "mode": either "basic" and "traditional"
+
+As of 2021-04-14, "mode" is a corse mechanism for affecting the display; there are a
```
Comment (andrewdotn) [inline CreeDictionary/CreeDictionary/display_options.py:6]:
```suggestion
As of 2021-04-14, "mode" is a coarse mechanism for affecting the display; there are
```

File: CreeDictionary/CreeDictionary/views.py [hunk 2]
```diff
@@ -236,6 +236,41 @@ def google_site_verification(request):
...
+
+    def post(self, request) -> HttpResponse:
+        mode = request.POST.get(DISPLAY_MODE_COOKIE)
+
+        # Tried to set to an unknown display mode
+        if mode not in DISPLAY_MODES:
+            return HttpResponse(status=HTTPStatus.BAD_REQUEST)
+
+        if who_asked_us := request.headers.get("Referer"):
+            # Force the browser to refersh the page that issued this request.
```
Comment (andrewdotn) [inline CreeDictionary/CreeDictionary/views.py:262]:
```suggestion
            # Force the browser to refresh the page that issued this request.
```

File: cypress/integration/search/header-display-options.spec.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
+/**
+ * Similar to ./result-header.spec.js -- displays a more deta
```
Comment (andrewdotn) [inline cypress/integration/search/header-display-options.spec.js:2]:
```suggestion
 * Similar to ./result-header.spec.js -- displays more data
```

File: src/css/styles.css [hunk 4]
```diff
@@ -597,6 +602,16 @@ a.multiple-recordings__action-button {
   cursor: help;
 }
 
+
+/**
+ * ELEMENT inflecional-class
```
Comment (andrewdotn) [inline src/css/styles.css:607]:
```suggestion
 * ELEMENT inflectional-class
```

**Review by andrewdotn — APPROVED** [review]
Looks ok. I know that in the templates themselves it always shows up as `display.mode`, but I have a vague sense that `display` may be too generic a term to use as a default context variable. Maybe `display_mode`? `display_options.mode`? If it does become an issue, we could always change it later.

**Comment by aarppe** [comment]
Would the pop-ups be in reverse?

That is, for the community mode, what you see initially is 1) the emojis and the 'like: X' text, and secondarily in the pop-up 2) the linguistic labels (long 'regular transitive animate verb' and short '(VTA-1)').

And for the linguistic mode, you'd initially see 1) the short linguistic label, e.g. VTA-1, and below that the full linguistic label but then secondarily in the pop-up 2) the example word case 'like: XXX' (but perhaps not the emoji).

Eventually I'd think we'd want to include the nêhiyawêwin labels in the mix.

**Pull request merged**